config/routes.rb

```diff
@@ -10,9 +10,11 @@
   resources :cats, only: [:index, :show]
   resources :order_cats, only: [:create]
   resources :orders, only: [:index, :show]
-  resources :cart_items, only: [:create, :delete, :show]
+  resources :cart_items, only: [:create, :show]
   get '/cart_items', to: "cart_items#show", as: "cart"
   patch '/cart_items', to: "cart_items#increment", as: "increment_cart_items"
+  put '/cart_items', to: "cart_items#decrement", as: "decrement_cart_items"
+  delete '/cart_items', to: "cart_items#delete"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.